dev/com.ibm.websphere.appserver.features/visibility/public/acmeCA-2.0/com.ibm.websphere.appserver.acmeCA-2.0.feature

```diff
@@ -6,10 +6,9 @@ IBM-ShortName: acmeCA-2.0
 Subsystem-Version: 2.0
 Subsystem-Name: Automatic Certificate Management Environment (ACME) Support 2.0
 -features=\
-  com.ibm.wsspi.appserver.webBundle-1.0
+  com.ibm.wsspi.appserver.webBundle-1.0,\
   com.ibm.websphere.appserver.appSecurity-1.0
 -bundles=\
   com.ibm.ws.security.acme; start-phase:=APPLICATION_EARLY
```

Comment:
@kristip17 is there a particular reason this bundle needs to be in start-phase `APPLICATION_EARLY`?

Reply:
Jim M's sample used application early to start talking to the ACME CA. If we continue with the same path, then yes. I'll added a note to #9867 to verify if we still need it.